tests/components/subjects/ModuleListItem.test.js

```diff
@@ -0,0 +1,18 @@
+import { shallow } from 'enzyme';
+import ModuleListItem from 'components/subjects/ModuleList';
+
+describe('ModuleListItem', () => {
+	it('should match previous snapshot', () => {
+		const sut = (<ModuleListItem id="abcd" name="abcd" description="dummy description" />);
```

Comment:
Test passes with warning:
Warning: Failed prop type: The prop `modules` is marked as required in `ModuleList`, but its value is `undefined`.
          at ModuleList (/app/src/components/subjects/ModuleList/ModuleList.jsx:14:23)